nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearchRecord.java

```diff
@@ -86,6 +96,16 @@
         resource = SystemResource.MEMORY,
         description = "The Batch of Records will be stored in memory until the bulk operation is performed.")
 public class PutElasticsearchRecord extends AbstractPutElasticsearch {
+    static final Relationship REL_FAILED_RECORDS = new Relationship.Builder()
+            .name("errors").description("If a \"Result Record Writer\" is set, any record that failed to process the way it was " +
+                    "configured will be sent to this relationship as part of a failed record set.")
+            .autoTerminateDefault(true).build();
+
+    static final Relationship REL_SUCCESSFUL_RECORDS = new Relationship.Builder()
```

Comment:
I understand the naming challenge here, since it was perhaps more descriptive to call the failed records relationship something like "record errors" and we don't want to break backwards compatibility, but I think the "success" vs "successful" is potentially too confusing.  I'd recommend calling this "successful records".

Reply:
I think the existing `success` would be better as `original` and `errors` as `error_records` - but as you say, that will break backwards compatibility

Happy to go with `auccessful_records` for the new relationship as a compromise